main.tex

```diff
@@ -357,13 +358,14 @@ \section*{Introduction}
 
 % What is our goal of this publication? And what is the Wnt signaling pathway?
 In this publication, we will identify and structure relevant information needed for the provenance of simulation studies and elucidate how a family of simulation models can be established through relating the models to each other.
+% elucidate?
 As a case study, we will concentrate on 19 simulation studies of the \wnt{}/\bcat{} signaling pathway.
 Among the different \wnt{} signaling pathways, the canonical \wnt{} or \wnt{}/\bcat{} signaling pathway is the most intensively studied one, \textit{in vitro}~\cite{MacDonald2009} as well as \textit{in silico}~\cite{Lloyd-Lewis2013}.
 This pathway is considered to be one of the key pathways in development and regeneration, including cell fate, cell proliferation, cell migration and adult homeostasis~\cite{Steinhart2018, Giles2003}.
 In deregulated forms, it is involved in human cancers and developmental disorders~\cite{Clevers2012, Logan2004}.
 
 Our case study refers to the \wnt{}/\bcat{} signaling pathway only, which we call \wnt{} throughout the text.
-We will present and use our web-based tool \webprov{} to store, present and query provenance information of these simulation studies.
+We will present and use our web-based tool \webprov{} to store, present and query provenance information from these simulation studies.
```

Comment:
Isn't it provenance information "of" a study rather then "from"? And do you have a suggestions to replace one "present"?

Reply:
I think "from" might be the better word since it indicates that the provenance information originated from the simulation studies. "of" also works but sounds a bit off to me.

To replace one "present", I would replace the second use of the word with display: "store, display and query"

Reply:
I went through the paper again and chose the following prepositions depending on the wording:
provenance information from a study
provenance of a simulation model
provenance of a study (meaning mainly the graph)
provenance graph of a study
What do you think?

Reply:
Those all sound great :)